homework/Classes_scope_ and_namespaces/task3.py

```diff
@@ -0,0 +1,63 @@
+
+
+CHANNELS = ["BBC", "Discovery", "TV1000"]
+
+
+class TVController:
+
+    current_channel = CHANNELS[0]   # default channel
```

Comment:
I'd suggest to initialize this attribute inside of `__init__` so that every instance of TVController would have a separate behaviour